app/assets/javascripts/discourse/app/components/sidebar/section-link.js

```diff
@@ -28,7 +28,11 @@ export default class SectionLink extends Component {
     let classNames = ["sidebar-section-link", "sidebar-row"];
 
     if (this.args.linkName) {
-      classNames.push(`sidebar-section-link-${this.args.linkName}`);
+      classNames.push(
+        `sidebar-section-link-${this.args.linkName
+          .replace(" ", "-")
```

Comment:
You actually need to use `replaceAll`, this just does one ;)

Reply:
great catch! Thank you :)